src/callbacks_stage/subcell_bounds_check_2d.jl

```diff
@@ -85,40 +84,48 @@
                                            idp_bounds_delta_local[key][stride_size])
     end
 
-    if save_errors
-        # Print to output file
-        open("$output_directory/deviations.txt", "a") do f
-            print(f, iter, ", ", time)
-            if local_minmax
-                for v in limiter.local_minmax_variables_cons
-                    v_string = string(v)
-                    print(f, ", ",
-                          idp_bounds_delta_local[Symbol(v_string, "_min")][stride_size],
-                          ", ",
-                          idp_bounds_delta_local[Symbol(v_string, "_max")][stride_size])
-                end
+    return nothing
+end
+
+@inline function save_bounds_check_errors(output_directory, u, time, iter, equations,
+                                          limiter::SubcellLimiterIDP)
+    (; local_minmax, positivity) = limiter
+    (; idp_bounds_delta_local) = limiter.cache
+
+    stride_size = div(128, sizeof(eltype(u))) # = n
```

Comment:
Why do you need `stride_size` here - isn't the output routine non-threaded?

If you _do_ need to keep this, you should add a comment again explaining why this is happening this way...

Reply:
Before, when calculating the maximum deviation over every thread, I save the maximum in the first used entry which has the index `stride_size`. See https://github.com/bennibolm/Trixi.jl/blob/cad2c0aadb3988ea0077fc1df802c611c2a971eb/src/callbacks_stage/subcell_bounds_check_2d.jl#L78-L85

Alternatively, I could save the maximum in the first entry. Then, I would write `[1]` instead of `[stride_size]`.

But even then, I need `stride_array` [at the end](https://github.com/bennibolm/Trixi.jl/blob/cad2c0aadb3988ea0077fc1df802c611c2a971eb/src/callbacks_stage/subcell_bounds_check_2d.jl#L124-L129) if I don't want to unnecessarily reset all entries.

Reply:
Solved in #1888.